manifest.php

```diff
@@ -28,7 +28,7 @@
     'label' => 'Back Office',
     'description' => 'Base for back-office extensions',
     'license' => 'GPL-2.0',
-    'version' => '5.1.0',
+    'version' => '5.1.1',
```

Comment:
`tao` version needs to be bumped in dependencies

Reply:
done thank you